### ISSUE TITLE
Separate website for mappers

### ISSUE THREAD

**Issue by ADepic** [osmlab/osm-planning#22]
## I am proposing...

... to make a separate website and vector map aimed specifically at mappers (and maybe developers too.)

### Prerequisites 

- Vector Map Tiles ([issue](https://github.com/openstreetmap/operations/issues/214))

### What will this enable that we can't do today?

Today, the main website it aimed at mappers & developers. However, this website is also the first thing you get when searching a openstreetmap related term ("openstreetmap" : [Google](https://www.google.com/search?hl=en&q=openstreetmap), [Duckduckgo](https://duckduckgo.com/?q=openstreetmap&t=ffab&ia=web), [Bing](https://www.bing.com/search?q=openstreetmap) - "What is openstreetmap": [2nd result Google](https://www.google.com/search?hl=en&q=what%20is%20openstreetmap), [4th result DuckDuckGo](https://duckduckgo.com/?q=what+is+openstreetmap%3F&t=ffab&ia=web), [doesn't appear in Bing](https://www.bing.com/search?q=what+is+openstreetmap&qs=n&form=QBRE&sp=-1&pq=undefined&sc=0-21&sk=&cvid=E127AA543F7A45588E7E45716E949675)). 

The problem with this is how a first impression of the project shows to users. [They see a raster map that doesn't seem to want to be used](https://github.com/openstreetmap/operations/issues/214), a relatively weak geo-coding engine ([example](https://gfycat.com/NiceImperturbableCatfish), [issue](https://github.com/openstreetmap/Nominatim/issues/759)) and if they somehow manage to get the descriptions of a address/POI/any data (three ways of doing this, Right Click > Query Features, Right Click > Show Address (only works for POI/addresses, not any random piece of osm data) and Map Layers > Enable Map Data, click on POI), they see a [bunch of confusing terminology aimed at mappers and no user friendly description](https://github.com/osmlab/osm-planning/issues/21).

If a separate map style and website was given to mappers (it could be called something like "mappers.openstreetmap.org"), the main website can solely be focused on giving a good impression and being user-friendly and very useable.

Not only this, but this mapper website will be aimed at mappers, meaning it can have a lot of cool features that are not currently on osm.org (will discuss below)

### Whom does this benefit?

Both Consumers of openstreetmap, developers, editors and potentially buisnesses too.

### Any drawbacks?

I see many problems and hurdles to get this done. The first is making a new map style, which means artwork and adding the features I mentioned. However, with vector maps this might be easier. For a example see Osmand - it shows the same maps in multiple different styles.

Another is the hosting. Maybe At small zooms, the map renders from openstreetmap-carto, and at higher zooms a vector map loads. This could help the hosting solution.

Another downside is that this might anger some mappers. They might see it as move by osm to appeal to the masses, or maybe seperating mappers from the crowd, or maybe hiding their hard work etc.. (this is purely speculation).

I think if something like this was implemented, it could be like the odbl change -  a bit controversial.

### Resources needed to build and maintain?

A new map style will be needed, based on vector maps. This will be a hosting challenge too. Apparently though, vector maps are a lot more compact then raster maps, which could be quite helpful.

### New features in the mapper oriented website.

The reason I would highly recommend vector tiles before doing this is that it could allow lots of powerful features in the website. For example on that website you should be able to hover on a node/way/outline of area to see who has last edited it, etc. 
That is just one idea. There are so much more - you could have notes load by default, option to show tags, groups to collorabate, maybe set tasks, a tasking manager for surveying - so much more.

These are just some of the things I could of the top of my head. If any of you could think of something else feel free to post that idea in a comment

### Any ideas for a project name?

mappers.openstreetmap.org

Website name maybe?

Otherwise can't think of a project name that isn't contraversial & related.
### Describe this project in a single emoji:

:+1:

**Comment by woodpeck** [comment]
I think the opposite should be done, the map needs to be removed from www.openstreetmap.org. If you go to www.openstreetmap.org, you should see something about the project - about our community, who we are, what we do, how you can get involved, etc., and then there should be links to various types of map - some aimed at mappers and maintained by us, and some aimed at various other target groups that may or may not be maintained by us.

Bit like e.g. www.openstreetmap.de (with a more modern & appealing gallery - the osm.de site is a couple years old and it shows).

**Comment by andrewharvey** [comment]
> I think the opposite should be done, the map needs to be removed from www.openstreetmap.org. If you go to www.openstreetmap.org, you should see something about the project - about our community, who we are, what we do, how you can get involved, etc., and then there should be links to various types of map - some aimed at mappers and maintained by us, and some aimed at various other target groups that may or may not be maintained by us.

I agree, make www.openstreetmap.org information about OSM, and map.openstreetmap.org the map.
